src/votekit/election_types.py

```diff
@@ -1,98 +1,125 @@
 from .profile import PreferenceProfile
 from .ballot import Ballot
-from .models import Outcome
+from .election_state import ElectionState
 from typing import Callable
 import random
 from fractions import Fraction
 from copy import deepcopy
 
-
+##We're not passing winner_votes to outcome right now
+# If we want to track which candidate each ballot is ultimately going to, we need to do more.
+# TODO:
+# 1. winner_votes to election_state is supposed to have all the ballots in the
+# initial profile that ended up going to any elected candidate.
```

Comment:
to this point i think we should store the initial profile in the STV class, just to have easy access to it. Also since the user has to pass a PreferenceProfile to initialize the class it seems weird to immediately store that in a separate object. It can be a hidden attribute if we don't want users calling it from the STV class
`
self.__profile = profile
`